index.html

```diff
@@ -29,26 +31,139 @@ <h4>Responsive Avatar</h4>
+        <div class="buttons">
+            <button class="secondary-btn">Cancel</button>
+            <button class="primary-btn">Save</button>
+        </div>
+    </div>
+    <h2>Badge</h2>
+    <p>Badges are used to highlight an item's status for quick recognition.</p>
+    <div>
+        <div>
+            <button class="primary-btn">Notification<span class="badge"> 5</span></button>
+        </div>
+        <div>
+            <img src="./images/avatar-img/006-user-5.svg" alt="avatar-48" class="avatar-48 badge-avatar">
+            <span><img src="./images/online.svg" alt="online" class="online-offline"></span>
+        </div>
+        <div>
+            <img src="./images/cart.svg" alt="add to cart" class="cart">
+           <span class=" cart-badge">4</span>
+        </div>
+    </div>
+    <h2>Card</h2>
+    <p>Card is used to display an item of the page.</p>
+    <div class="card-div">
+        <div class="card">
+            <img src="./images/plant-card.jpg" alt="plants" class="card-img">
+            <p>Leafy House Plant</p>
+            <p>₹ 999</p>
+            <button class="primary-btn">Add to Cart</button>
+        </div>
+        <div class="category-card">
+            <img
+              src="https://cdn.shopify.com/s/files/1/0535/8049/7061/files/banner6.1.jpg?v=1612837951"
+              alt="procelain-pots" class="category-img"
+            />
+            <h4 class="card-data">NAME</h4>
+            <p class="card-data">Lorem ipsum dolor sit amet consectetur adipisicing elit. Consectetur in repellat officiis enim! Mollitia vero eum quod voluptatum aut harum? Adipisci sunt dolores nesciunt architecto minima dolor id corrupti a.</p>
+            <button class="primary-btn">View More</button>
+          </div>
+        
+    </div>
+    <h2>Search Bar</h2>
+    <p>Search Bar is used to search an item.</p>
+    <div>
+        <div class="search-bar">
+            <input type="search" id="search" class="input-fields primary search-input" placeholder="Search">
+            <img src="./images/fa fa search.png" alt="search-bar" class="search-icon">
+             
+        </div>
+    </div>
+    <h2>Tooltip</h2>
+    <p>Tooltip is a small pop-up box that appears when the user hovers an element.</p>
+    <div class="tooltip">
+        <p>Hover over me</p>
+        <span class="tooltiptext">Save</span>
+    </div>
+    <h2>Toast/Snackbar</h2>
+    <p>Toast/Snackbars are often used as tooltips/popups to show a message at the bottom of the screen.</p>
+    <div>
+        <button onclick="showToast()" class="primary-btn">Click Me!</button>
+        <div id="toast">Message</div>
+    </div>
+    <footer class="footer">
+        <div class="footer-header">See Code on:</div>
+        <ul>
+            <li class="list-inline">
+                <a target="_blank" class="link" href="https://github.com/vids12/Commponent-Library">Github</a>
+            </li>
+        </ul>
+    </footer>
+    <script>
+        function showToast(){
```

Comment:
putting js in different file will save your life if you are working on any component and i starts growing in this case creating external js instead of doing internal or inline js .

Reply:
Sure!! Will make the changes